src/Raven.Server/Documents/Subscriptions/SubscriptionConnectionsState.cs

```diff
@@ -634,5 +634,31 @@ private SubscriptionConnectionsState(DocumentsStorage storage, SubscriptionState
         }
 
         public static SubscriptionConnectionsState CreateDummyState(DocumentsStorage storage, SubscriptionState state) => new(storage, state);
+
+        private long _lastNoopAckTicks;
+        private Task _lastNoopAckTask = Task.CompletedTask;
+        public Task SendNoopAckForConcurrentSubscription()
+        {
```

Comment:
handle concurrent and regular subscription here, would prefer to have 1 if here, than in 2 places same 'DocumentDatabase.SubscriptionStorage.AcknowledgeBatchProcessed' code

Reply:
done